util/reggen/reg_block.py

```diff
@@ -382,9 +382,16 @@ def make_intr_regs(self, interrupts: Sequence[Signal]) -> None:
                                  'rw1c',
                                  'hrw',
                                  False,
-                                 # intr_state csr is affected by writes to
-                                 # other csrs - skip write-check
-                                 ["excl:CsrNonInitTests:CsrExclWriteCheck"])
+                                 # see issue#6888 for further details.
+                                 #
+                                 # The rom_ctrl<->kmac interaction right after
+                                 # reset causes kmac.intr_state to become
+                                 # updated before CSR tests can start at the
+                                 # chip leve, leading to unexpected failures.
+                                 #
+                                 # As per #6888, exclude intr_state from CSR
+                                 # checks to avoid this issue.
```

Comment:
IMO we could keep this a bit generic - "POR initialization routines / writes to other CSRs could unpredictably set some of the fields in intr_state."  It is not a far-fetched thought, so there is no need to justify with a bug number. WDYT?

Reply:
I agree with having a more generic description for this, but I feel it may be good to link to the bug just to show an actual example of some scenario that requires this particular fix, wdyt?

Reply:
In that case "POR initialization routines (for example, #6888) / writes to other CSRs could unpredictably set some of the fields in intr_state.".

Reply:
sg - I"ve updated the comment along similar lines, ptal